doc/examples/applications/plot_fluorescence_nuclear_envelope.py

```diff
@@ -0,0 +1,217 @@
+# and foreground with Otsu's method: We get a binary image.
+
+smooth = filters.gaussian(image_t_0_channel_0, sigma=1.5)
+
+thresh_value = filters.threshold_otsu(smooth)
+thresh = smooth > thresh_value
+
+fill = ndi.binary_fill_holes(thresh)
+
+#####################################################################
+# Following the original workflow, let us remove objects which touch the image
+# border. Here, we can see part of another nucleus in the bottom right-hand
+# corner.
+
+clear = segmentation.clear_border(fill)
+label = measure.label(clear)
+
+expand = segmentation.expand_labels(label, distance=1)
+
+erode = morphology.erosion(label)
+
+mask = expand - erode
+
+_, ax = plt.subplots(2, 4, figsize=(12, 6), sharey=True)
+
+ax[0, 0].imshow(image_t_0_channel_0, cmap=plt.cm.gray)
+ax[0, 0].set_title('a) Raw')
+
+ax[0, 1].imshow(smooth, cmap=plt.cm.gray)
+ax[0, 1].set_title('b) Blur')
+
+ax[0, 2].imshow(thresh, cmap=plt.cm.gray)
+ax[0, 2].set_title('c) Threshold')
+
+ax[0, 3].imshow(fill, cmap=plt.cm.gray)
+ax[0, 3].set_title('c-1) Fill in')
+
+ax[1, 0].imshow(label, cmap=plt.cm.gray)
+ax[1, 0].set_title('c-2) Keep one nucleus')
+
+ax[1, 1].imshow(expand, cmap=plt.cm.gray)
+ax[1, 1].set_title('d) Expand')
+
+ax[1, 2].imshow(erode, cmap=plt.cm.gray)
+ax[1, 2].set_title('e) Erode')
+
+ax[1, 3].imshow(mask, cmap=plt.cm.gray)
+ax[1, 3].set_title('f) Nucleus Rim')
+
+for a in ax.ravel():
+    a.axis('off')
+
+#####################################################################
+# Apply the segmented rim as a mask
+# =================================
+# Now that we have segmented the nuclear membrane in the first channel, we use
+# it as a mask to measure the intensity in the second channel.
+
+image_t_0_channel_1 = image_sequence[0, 1, :, :]
+selection = np.where(mask, image_t_0_channel_1, 0)
+
+_, ax = plt.subplots(1, 2, figsize=(12, 6), sharey=True)
+
+ax[0].imshow(image_t_0_channel_1)
+ax[0].set_title('Second channel (raw)')
+ax[0].axis('off')
+
+ax[1].imshow(selection)
+ax[1].set_title('Selection')
+ax[1].axis('off')
+
+#####################################################################
+# Measure the total intensity
+# ===========================
+# The mean intensity is readily available as a region property in a labeled
+# image.
+
+props = measure.regionprops_table(
+    mask,
+    intensity_image=image_t_0_channel_1,
+    properties=('label', 'area', 'intensity_mean')
+)
+
+#####################################################################
+# We may want to check that the value for the total intensity
+
+selection.sum()
+
+#####################################################################
+# can be recovered from:
+
+props['area'] * props['intensity_mean']
+
+#####################################################################
+# Iterate the measurement for each time point
+# ===========================================
+# Let us write a function for the first processing step (segmentation of
+# nucleus rim).
```

Comment:
Personally, and this is a personal preference and happy to be overridden, I would make a 3D volume where each timepoint has a different segment value. We can then use `regionprops_table` directly with `label` and `intensity_mean` and plot them directly.

To do this "cleanly" we'd probably need to use a "flat" structuring element and only use dilate/erode instead of expand_labels:

```
footprint = np.stack((np.zeros((3, 3)), np.ones((3, 3)), np.zeros((3, 3)))
```

I'm not sure it's better, I just like avoiding the for-loop computing a single regionprop each time. 😂

Reply:
@jni @grlee77 please see my nD porting at 5002298d002d (there's just Otsu's threshold which I couldn't use in nD).

Reply:
> there's just Otsu's threshold which I couldn't use in nD

More specifically, you mean that you want a different threshold per timepoint as opposed to a global one over all time points? The function is nD so you could get a single threshold in one call, but you need to call it like you have it now if you want a unique threshold per timepoint.

Reply:
You could also reshape the data to 2D where timepoints is the first axis and then call using NumPy's apply_along_axis. It may be clearer to just leave it as you have it now, though?

```Python
np.apply_along_axis(threshold_otsu, 1, smooth_seq.reshape((a.shape[0], -1)))
```

Reply:
> > there's just Otsu's threshold which I couldn't use in nD
> 
> More specifically, you mean that you want a different threshold per timepoint as opposed to a global one over all time points? The function is nD so you could get a single threshold in one call, but you need to call it like you have it now if you want a unique threshold per timepoint.

Yes, exactly. Sorry, I should have been more specific in my assertion. I just double-checked the original workflow again and, to be faithful to it, we want to compute a threshold for each timepoint.

Reply:
> You could also reshape the data to 2D where timepoints is the first axis and then call using NumPy's apply_along_axis. It may be clearer to just leave it as you have it now, though?
> 
> ```python
> np.apply_along_axis(threshold_otsu, 1, smooth_seq.reshape((smooth_seq.shape[0], -1)))
> ```

I like it! I have added it for pedagogical purposes: f3bb06658d4a5.